src/js/components/Meter/Circle.js

```diff
@@ -147,12 +147,17 @@ const Circle = forwardRef((props, ref) => {
     viewBoxHeight = '100%';
   } else viewBoxHeight = width;
 
+  let viewBoxSize;
```

Comment:
It doesn't look like we're ever setting `viewBoxSize`?

Reply:
yes sorry, I committed the wrong change. 

I think the problem here was before the `viewBox={0 0 ${width} ${width}}`
This was changed so that when `size="full"` it was set to '100%' this should only be applied to the height

Reply:
also saw that when `size="full"` for `semimeter` was not working correctly so I added that check